frontend/benefit/handler/src/components/applicationForm/formContent/companySection/deMinimisAid/DeMinimisAidForm.tsx

```diff
@@ -36,6 +37,36 @@ const DeMinimisAidForm: React.FC<DeMinimisAidFormProps> = ({ data }) => {
   } = useDeminimisAid(data);
   const theme = useTheme();
 
+  const { setUnfinishedDeMinimisAidRow } = React.useContext(DeMinimisContext);
+
+  const validateDeMinimisRow = (): boolean =>
+    !(
+      formik.values.granter &&
+      formik.values.amount &&
+      formik.values.grantedAt
+    ) ||
+    !formik.isValid ||
+    sumBy(grants, 'amount') > MAX_DEMINIMIS_AID_TOTAL_AMOUNT;
+
+  /**
+   * Only empty inputs are consired complete because row addition clears them.
+   */
+  const isFormInputIncomplete: () => boolean = useCallback(
+    (): boolean =>
+      formik.values.granter?.length > 0 ||
+      formik.values.amount?.length > 0 ||
+      formik.values.grantedAt?.length > 0,
+    [formik.values]
+  );
+
+  useEffect(() => {
+    if (isFormInputIncomplete()) {
+      setUnfinishedDeMinimisAidRow(true);
+    } else {
+      setUnfinishedDeMinimisAidRow(false);
+    }
```

Comment:
This effectively just equals to `setUnfinishedDeMinimisAidRow(isFormInputIncomplete())`, the if-else is redundant.